### PR TITLE
Add pom.xml to akka-sample-cluster-java

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: akka-sample-cluster-java/pom.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      <id>2.12.4</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <properties>
+        <scalaBinaryVersion>2.12</scalaBinaryVersion>
+      </properties>
+    </profile>
+    <profile>
+      <id>2.11.8</id>
```
Comment (patriknw) [inline akka-sample-cluster-java/pom.xml]: I don't see a reason for providing two alternatives here. Java dev shouldn't care, and we can make the build simpler by only using 2.12

Comment (johanandren) [reply akka-sample-cluster-java/pom.xml]: I agree, we should remove it from the existing poms as well, I guess this was just copy-pasta from there?

Comment (patriknw) [reply akka-sample-cluster-java/pom.xml]: shall we add gradle also, while we are at it?

Comment (johanandren) [reply akka-sample-cluster-java/pom.xml]: That would be nice, but maybe make that a separate issue to not overwhelm @raboof with somewhat unrelated work in this PR

**Pull request merged**